package.json

```diff
@@ -39,12 +39,13 @@
     "dist"
   ],
   "peerDependencies": {
-    "vue": "^2.7.16"
+    "vue": "3"
   },
   "dependencies": {
     "@nextcloud/axios": "^2.4.0",
     "@nextcloud/router": "^3.0.0",
-    "@nextcloud/vue": "^8.11.2",
+    "@nextcloud/vue": "^9.0.0-alpha.2",
+    "@vue/compiler-sfc": "^3.4.23",
```

Comment:
`@vue/compiler-sfc` is not needed as a dependency, it is a sub dependency of `vue` in Vue 3.2+